src/Jackett.Common/Indexers/FileListApi.cs

```diff
@@ -0,0 +1,216 @@
+        };
+
+        private string ApiUrl => SiteLink + "api.php";
+
+        private static string Instructions => "Go into your filelist profile and copy the passkey.";
+
+        private new ConfigurationDataFileListApi configData
+        {
+            get => (ConfigurationDataFileListApi)base.configData;
+            set => base.configData = value;
+        }
+
+        public FileListApi(IIndexerConfigurationService configService, WebClient wc, Logger l, IProtectionService ps)
+            : base(name: "FileListApi",
+                description: "The best Romanian site.",
+                link: "https://filelist.ro/",
+                caps: TorznabUtil.CreateDefaultTorznabTVCaps(),
+                configService: configService,
+                client: wc,
+                logger: l,
+                p: ps,
+                configData: new ConfigurationDataFileListApi(Instructions))
+        {
+            Encoding = Encoding.UTF8;
+            Language = "ro-ro";
+            Type = "private";
+
+            TorznabCaps.SupportsImdbMovieSearch = true;
+
+            AddCategoryMapping(24, TorznabCatType.TVAnime, "Anime");
+            AddCategoryMapping(11, TorznabCatType.Audio, "Audio");
+            AddCategoryMapping(15, TorznabCatType.TV, "Desene");
+            AddCategoryMapping(18, TorznabCatType.Other, "Diverse");
+            AddCategoryMapping(16, TorznabCatType.Books, "Docs");
+            AddCategoryMapping(25, TorznabCatType.Movies3D, "Filme 3D");
+            AddCategoryMapping(6, TorznabCatType.MoviesHD, "Filme 4K");
+            AddCategoryMapping(26, TorznabCatType.MoviesBluRay, "Filme 4K Blu-Ray");
+            AddCategoryMapping(20, TorznabCatType.MoviesBluRay, "Filme Blu-Ray");
+            AddCategoryMapping(2, TorznabCatType.MoviesDVD, "Filme DVD");
+            AddCategoryMapping(3, TorznabCatType.MoviesForeign, "Filme DVD-RO");
+            AddCategoryMapping(4, TorznabCatType.MoviesHD, "Filme HD");
+            AddCategoryMapping(19, TorznabCatType.MoviesForeign, "Filme HD-RO");
+            AddCategoryMapping(1, TorznabCatType.MoviesSD, "Filme SD");
+            AddCategoryMapping(5, TorznabCatType.AudioLossless, "FLAC");
+            AddCategoryMapping(10, TorznabCatType.Console, "Jocuri Console");
+            AddCategoryMapping(9, TorznabCatType.PCGames, "Jocuri PC");
+            AddCategoryMapping(17, TorznabCatType.PC, "Linux");
+            AddCategoryMapping(22, TorznabCatType.PCPhoneOther, "Mobile");
+            AddCategoryMapping(8, TorznabCatType.PC, "Programe");
+            AddCategoryMapping(27, TorznabCatType.TVHD, "Seriale 4K");
+            AddCategoryMapping(21, TorznabCatType.TVHD, "Seriale HD");
+            AddCategoryMapping(23, TorznabCatType.TVSD, "Seriale SD");
+            AddCategoryMapping(13, TorznabCatType.TVSport, "Sport");
+            AddCategoryMapping(12, TorznabCatType.AudioVideo, "Videoclip");
+            AddCategoryMapping(7, TorznabCatType.XXX, "XXX");
```

Comment:
Missing category `14 => TV`  ?

Reply:
I feel like these could also be better sorted. Either bat category number, or by Category Type/Subtype.

Reply:
The api description is wrong. There is no category with id = 14.
I'll sort the categories by id.